Extensions/ArtifactEngine/Providers/filesystemProvider.ts

```diff
@@ -65,6 +69,9 @@ export class FilesystemProvider implements models.IArtifactProvider {
                     (error) => {
                         reject(error);
                     });
+                outputStream.on("finish", () => {
+                    this.artifactItemStore.updateFileSize(item, outputStream.bytesWritten);
```

Comment:
looks like fileSize is populated only for fileSystemProvider & downloadSize for webProvider. 

why not call this just downloadSize and populate it for both fileSystemProvider & webProvider?

Reply:
fileSize represents total size of file written which is common for both vsts drop and file share.

downloadSize is the data sent by server (in case of vsts drop via webProvider). Currently we are not tracking it for fileShare drops.

Reply:
got it